### DISCUSSION TITLE
Error Detect Inf/NaN in volume output. abortting.

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#737] [Q&A] [answered]
I have run a example made by myself, detailed documentation can be found at the link below
https://github.com/buzhiqiuyuebai/Error-with-Detected-Inf-NaN-volume
I have encouter this problem, what's the reason of this problem, This problem seems to have arisen before,such as  https://github.com/SeisSol/SeisSol/discussions/695
，Is it caused by too small mesh or mesh quality?  Or maybe other reason?
![image](https://user-images.githubusercontent.com/114474894/203556260-9dfe29f0-a494-46e6-bba9-8db2ae4bea40.png)

By the way, if I want to modify the wavelength of the point source, do I modify `T` in the `generate_LOH_source_type50.py`(https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/generate_LOH_source_type50.py) file or modify `FreqCentral=2.5 FreqRatio=100`  in the parameters.par (https://github.com/SeisSol/Examples/blob/master/WP2_LOH3/parameters.par) file? or modify other files?

![image](https://user-images.githubusercontent.com/114474894/203562053-8efc9ad1-7d26-4fc2-a40c-a7f96088e27f.png)
![image](https://user-images.githubusercontent.com/114474894/203562291-06d5f69b-9eef-4134-a072-caeee6e8920c.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
try ```CFL = 0.35```, to my experience ```CFL=0.5``` is too large for 0.6 (probably 0.45 is enough but 0.35 puts you on the safe side.

**Reply by buzhiqiuyuebai** [reply]
> try `CFL = 0.35`, to my experience `CFL=0.5` is too large for 0.6 (probably 0.45 is enough but 0.35 puts you on the safe side.

But recently I have  encountered this problem again, and CFL=0.35 still does not work, I think it cansed by the mesh quality, Is there any way to locate the problematic grid?

**Comment by ravil-mobile** [comment]
@buzhiqiuyuebai, did you used CPUs or GPUs? If GPUs, can you try to run this on CPUs?

**Reply by Thomas-Ulrich** [reply]
CPU given the exe name

**Reply by buzhiqiuyuebai** [reply]
> @buzhiqiuyuebai, did you used CPUs or GPUs? If GPUs, can you try to run this on CPUs?

Thank you, I used CPU.

**Reply by buzhiqiuyuebai** [reply]
> CPU given the exe name
This is CPU configuration,
![image](https://user-images.githubusercontent.com/114474894/203566316-c0cb29c0-246c-42af-a7a8-cd0c4f57fd29.png)
This is the executable pragrom.
![image](https://user-images.githubusercontent.com/114474894/203567057-4400a902-57ad-4729-aaa4-9cf3f75fc7f5.png)

**Comment by Thomas-Ulrich** [comment]
https://seissol.readthedocs.io/en/latest/attenuation.html?highlight=FreqCentral#freqcentral-and-freqratio

**Reply by buzhiqiuyuebai** [reply]
> https://seissol.readthedocs.io/en/latest/attenuation.html?highlight=FreqCentral#freqcentral-and-freqratio

OK，Thank you.

**Reply by buzhiqiuyuebai** [reply]
> https://seissol.readthedocs.io/en/latest/attenuation.html?highlight=FreqCentral#freqcentral-and-freqratio

It can work smoothly， but the result donen't seem to be good, as the picture below, I set the left surface as `free surface`, A little blurry waveform can be seen on this surface， is it is caused by the model is too small but I didn't modify the wavelength？ Can I get a good looking waveform like the one shown below by modifying FreqCentral? Besides, does FreqRatio need to be modified?  Thanks in advance!
![image](https://user-images.githubusercontent.com/114474894/204077203-718dd5b0-37f4-4165-b455-749f2c8f77e4.png)

![image](https://user-images.githubusercontent.com/114474894/204077070-a05ebfa4-2592-47de-be58-c2049a480639.png)
